vassal-app/src/main/java/VASSAL/tools/imports/adc2/ADC2Module.java

```diff
@@ -744,7 +745,7 @@ private String getValueAsString(int index) {
         ++length;
         mask >>= 8;
       }
-      return new String(b, 0, length);
+      return new String(b, 0, length, StandardCharsets.UTF_8);
```

Comment:
This method needs to be checked for what the range of values `b` can hold is. My suspicion is that the range is printable ASCII, given the age of AdC2.

Reply:
Change to `StandardCharsets.US_ASCII`? Would that make a difference as UTF-8 is backwards-compatible with ASCII?

Reply:
Having read through the code, none of the bytes can be set outside the range [0x20,0x7f), so I'm fairly sure it's pure ASCII. I'd set it to StandardCharsets.US_ASCII to be explicit.